tests/sim_db_file_size_test.py

```diff
@@ -0,0 +1,27 @@
+"""test sim_db_file
+
+:copyright: Copyright (c) 2024 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+
+
+max_bytes = 10000
+
+
+def setup_module(module):
+    import os
+
+    os.environ.update(
+        SIREPO_JOB_MAX_MESSAGE_BYTES=f"{max_bytes}",
```

Comment:
Since this is quite large, you could combine this case in sim_db_file_test.py. Using an fstring here is a bit less clear to me than `str(max_bytes)`. I have to inspect the f string to see if anything is special with "str" I know it is simply converting a value to a string.

If you feel adventurous you could modify srunit_servers.sim_db_file to support fc_args (like it already does for api_and_supervisor) and then simplify the test to support passing args. See for example status_test, since sim_db_file fixture already sets the configuration, you could augment it. Alternatively, you could just set sunit_servers config to be limited and export the value of max_size.

Reply:
[5d923a5](https://github.com/radiasoft/sirepo/pull/6764/commits/5d923a5f715fd0a89a1b243f026eb20bfef7b1dd)

Reply:
I tried adventuring but just got confused. Can it be deferred to a future issue?